### PR TITLE
login screen changes

### PR DESCRIPTION
**Pull request by firnas16**
simulator_screenshot_5BC41752-6740-4567-A019-72B3A1BF2DE2

### CODE REVIEW

File: lib/translation/resources/en.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
 export default {
   welcome: 'Welcome',
+  lets_Get_Started: 'Let’s get started!',
```
Comment (Ganesh-softsuave) [inline lib/translation/resources/en.ts:3]: Please use key as lower case

File: src/screens/Login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import {testID} from '@src/utils/TestUtils';
+
+function Login(): JSX.Element {
+  const {t: translate} = useTranslation();
+  const themeContext: ThemeContextType = useThemeContext();
+  const getStyles: LoginScreenStyleSheet = styles(themeContext.color);
+  const {handleNumberChange, onSubmit, control, handleSubmit, isValid} =
+    ViewModel();
+
+  console.log('sssaa>>>');
```
Comment (Ganesh-softsuave) [inline src/screens/Login/index.tsx:24]: Please remove logs

File: src/screens/Login/index.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {View, Image, TextInput} from 'react-native';
+import {ThemeContextType} from '@src/context/types';
+import {useThemeContext} from '@src/context/themeContext';
+import styles from './styles';
+import {LoginScreenStyleSheet} from './types';
+import CustomButton from '@src/components/CustomButton';
+import ViewModel from './ViewModal';
+import {useTranslation} from 'react-i18next';
```
Comment (Ganesh-softsuave) [inline src/screens/Login/index.tsx:9]: Please use import statement orderwise

File: src/screens/Login/index.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  return (
+    <View style={getStyles.mainContainer}>
+      <CustomText style={getStyles.startText}>
+        {translate('lets_Get_Started')}
+      </CustomText>
+      <CustomText style={getStyles.enterText}>
+        {translate('enter_Your_Mobile')}
+      </CustomText>
+      <Controller
+        name={'phoneNumber'}
```
Comment (Ganesh-softsuave) [inline src/screens/Login/index.tsx:35]: Please create Separate component for Phone number text input

File: src/screens/Login/ViewModal.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import {useForm} from 'react-hook-form';
+function ViewModel() {
+  const {
+    control,
+    handleSubmit,
+    formState: {isValid},
+  } = useForm();
+
+  function handleNumberChange(input: string) {
```
Comment (Ganesh-softsuave) [inline src/screens/Login/ViewModal.ts:9]: Please move all this logic to Phonenumber component

File: src/screens/Login/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    mainContainer: {
+      flex: 1,
+      backgroundColor: ThemeColors.radicalRed,
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    startText: {
+      width: pixelSizeHorizontal(170),
+      height: pixelSizeVertical(56),
+      right: 80,
```
Comment (Ganesh-softsuave) [inline src/screens/Login/styles.tsx:18]: Please use pixelSizeHorizontal for all over the component

**Review by Ganesh-softsuave — CHANGES_REQUESTED** [review]

File: src/screens/Home/index.tsx [hunk 7]
```diff
@@ -4,14 +4,16 @@ import {ThemeContextType} from '@src/context/types';
 import {useThemeContext} from '@src/context/themeContext';
 import styles from './styles';
 import {HomeScreenStyleSheet} from './types';
+import Login from '../Login';
```
Comment (BhavaniMM) [inline src/screens/Home/index.tsx:7]: please import absolute path

**Review by Ganesh-softsuave — APPROVED** [review]

**Review by dilipkrishna97 — APPROVED** [review]

**Pull request merged**